{"version":1,"title":"general protection fault in call_timer_fn (2)","id":"6c95f16e4333a447aca35578d47da18e962816c6","discussions":["https://groups.google.com/d/msgid/syzkaller-upstream-moderation/68b55de1.a00a0220.1337b0.0040.GAE@google.com"],"crashes":[{"title":"general protection fault in call_timer_fn","kernel-config":"/text?tag=KernelConfig\u0026x=c321f33e4545e2a1","kernel-source-git":"https://git.kernel.org/pub/scm/linux/kernel/git/bpf/bpf-next.git/log/?id=d3abefe897408718799ae3bd06295b89b870a38e","kernel-source-commit":"d3abefe897408718799ae3bd06295b89b870a38e","syzkaller-git":"https://github.com/google/syzkaller/commits/e12e5ba469cd05aa76330394486b1f2a24dd4ef6","syzkaller-commit":"e12e5ba469cd05aa76330394486b1f2a24dd4ef6","crash-report-link":"/text?tag=CrashReport\u0026x=15e6e634580000"}]}